README.md

```diff
@@ -25,7 +25,7 @@ https://hawkbit.eu-gb.mybluemix.net/UI/
 
 # Compile, Run and Getting Started
 
-We are not providing an off the shelf installation ready hawkBit update server. However, we recommend to check out the [Example Application](examples/hawkbit-example-app) for a runtime ready Spring Boot based update server that is empowered by hawkBit. In addition we have [guide](https://github.com/eclipse/hawkbit/wiki/Run-hawkBit) for setting up a complete landscape.
+We are not providing an off the shelf installation ready hawkBit update server. However, we recommend to check out the [Example Application](examples/hawkbit-example-app) for a runtime ready Spring Boot based update server that is empowered by hawkBit. In addition we have [guide](http://www.eclipse.org/hawkbit/documentation/guide/runhawkbit.html) for setting up a complete landscape.
```

Comment:
> However, we recommend to check out the [Example Application](examples/hawkbit-example-app)

I guess we would write here that we do offer a hawkbit-runtime update-server for easy to startup an hawkBit server, ready to use, which can be customized by your needs, right?